File: fixtures/string-concat-in-loop/negative/basic.py
def sum_numbers(items):
    n = 0
    for x in items:
        n += x
    return n


def build_once(items):
    s = ""
    s += "prefix"
    for x in items:
        do_thing(x)
    return s


def build_in_nested_function(items):
    for x in items:
        def render():
            t = ""
            t += "z"
            return t

        render()

File: fixtures/string-concat-in-loop/negative/basic.ts
function sumNumbers(items: number[]): number {
  let n = 0;
  for (const x of items) {
    n += x;
  }
  return n;
}

function buildOnce(items: string[]): string {
  let s = "";
  s += "prefix";
  for (const x of items) {
    doThing(x);
  }
  return s;
}

function buildInNestedFunction(items: string[]): void {
  for (const x of items) {
    const render = (): string => {
      let t = "";
      t += "z";
      return t;
    };
    render();
  }
}
